easybuild/easyblocks/generic/intelbase.py

```diff
@@ -206,32 +207,33 @@ def prepare_step(self):
         """Custom prepare step for IntelBase. Set up the license"""
         super(IntelBase, self).prepare_step()
 
-        default_lic_env_var = 'INTEL_LICENSE_FILE'
-        lic_specs, self.license_env_var = find_flexlm_license(custom_env_vars=[default_lic_env_var],
-                                                              lic_specs=[self.cfg['license_file']])
+        if self.cfg['licensed']:
```

Comment:
ideally, we would have an equivalent named argument to `prepare_step`, which we can leverage from easyblocks deriving from `IntelBase`:

something like this (logic may be a bit off, since the point of this is to avoid having to use `requires_runtime_license = False` in all future `imkl` easyconfigs):

```python
def prepare_step(self, requires_runtime_license=True):
    ...
    if self.cfg['requires_runtime_license'] or requires_runtime_license:
        ...
```